### ISSUE TITLE
Recent changes in Phonegap/imagesloaded - updates to Readme?

### ISSUE THREAD

**Issue by notacouch** [chrisben/imgcache.js#15]
Using PhoneGap 3.0 implementation of features/plugins have changed.

For `device.*` see: [Device Plugin](http://docs.phonegap.com/en/edge/cordova_device_device.md.html#Device)

For the File API I think you need both the File and File Transfer plugins, definitely the latter. see: [File Plugin](http://docs.phonegap.com/en/edge/cordova_file_file.md.html#File)

Strangely enough the starting package still has this in its base config.xml:
`<feature name="http://api.phonegap.com/1.0/device" />`

but despite no mention of iOS in the phonegap docs above you will need to add the following to your iOS' config.xml for the Device plugin:
`<feature name="Device">
         <param name="ios-package" value="CDVDevice" />
     </feature>`

As for imagesloaded it's now in 3.x and seems to be totally different. I am personally using ChocolateChip-UI over jQuery so I had to port methods over one by one for compatibility, turns out I was porting the wrong version haha. Thank God for the examples in the repo!

Here's a link to imagesloaded's last tag of 2.x:

[imagesloaded 2.1.2](https://github.com/desandro/imagesloaded/tree/v2.1.2)

TL;DR:

You may want to update links in your readme to phonegap/cordova [Device](http://docs.phonegap.com/en/edge/cordova_device_device.md.html#Device) & [File Plugins](http://docs.phonegap.com/en/edge/cordova_file_file.md.html#File) and to [imagesloaded](https://github.com/desandro/imagesloaded/tree/v2.1.2).

Testing out ImgCache at long last...
`ImgCache.clear()` in README should also read `ImgCache.clearCache()`

Okay time to go to town with this I hope.

Thank you! =)
